src/main/java/info/wzieleziecki/Main.java

```diff
@@ -0,0 +1,33 @@
+package info.wzieleziecki;
+
+import org.apache.log4j.Logger;
+
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        final Logger logger = Logger.getLogger(Main.class);
+
+        String startWord = "ruby";
+        String endWord = "code";
+
+        if (startWord.length() != endWord.length()) {
+            logger.info("Words shoud contain the same number of letters");
+            System.exit(1);
+        }
+
+        Word word = new Word();
+
+        List<String> words = word.getWordsWithDefinedLetterNumber(startWord.length());
+        Graph graph = new Graph(words);
+        try {
+            List<String> path = graph.findShortestPath(startWord, endWord);
+            logger.info("Solution for: " + startWord + " to " + endWord + " - " +  path);
+        } catch (NullPointerException n) {
+            logger.info("No solution found. " + n);
```

Comment:
Nie pisz `n,` Pisz `n`.
Ta metoda jest do tego przygotowana aby otrzymywać aprametr jako exception :)

Ponadto generalnie powinno isę jak najmniej używać exceptionów o ile to możliwe. Skoro wykonujesz jedną operację, to prawdopodobnie da się ją o'ifować

Reply:
Zastosowałem